src/reanimated2/js-reanimated/index.ts

```diff
@@ -2,26 +2,32 @@ import JSReanimated from './JSReanimated';
 import { AnimatedStyle, StyleProps } from '../commonTypes';
 import { isWeb } from '../PlatformChecker';
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I've also had similar warnings in VSCode but for some reason our CI doesn't complain about it. Can we modify our workflows so that it's checked?

Reply:
It does complain about it but considers it a warning, not an error. I don't think we want to consider it as an error since we have many of those.